### PR TITLE
Update Lucid Aggregates docs

### PR DESCRIPTION
**Pull request by benallfree**

### CODE REVIEW

File: 07-lucid/01-model.adoc [hunk 1]
```diff
@@ -353,6 +353,47 @@ const User = use('App/Models/User')
...
+All helpers accept a column name to be used for aggregation. When possible Lucid will choose a default for the column name. Some methods, such as `sum()`, require a column name.
+
+==== rowsCount(columnName = '*')
+Return a count of records in a given result set.
+
+[source, js]
+----
+const User = use('App/Models/User')
+
+# Return count of all records in table
```
Comment (RomainLanz) [inline 07-lucid/01-model.adoc]: This isn't valid comment marks. It must be `//`.

File: 07-lucid/01-model.adoc [hunk 2]
```diff
@@ -353,6 +353,47 @@ const User = use('App/Models/User')
...
+await User.rowsCount()
+
+# Return count of selected records
+await User.query().rowsCount()
```
Comment (RomainLanz) [inline 07-lucid/01-model.adoc]: I think we should add a `where` clause to this example so people will understand it better.

**Comment by thetutlage** [comment]
I like the way you have explained. Happy to merge whenever you want

**Comment by benallfree** [comment]
Ok I will write the rest of the aggregates in `adonis-lucid` and update these docs as a joint PR.

**Comment by benallfree** [comment]
I can't use `count()`, `avg()`, `sum()`, `min()`, and `max()` because they will collide with Knex methods. Should I prefix everything with `rowsCount()`, `rowsAvg()`, `rowsSum()`, `rowsMin()`, and `rowsMax()`?

Other ideas...

`countAndFetch()`

`lcount()` (for Lucid)

Or we could take over the methods and proxy the Knex methods: `kcount()`, `kavg()`, `ksum()`, `kmin()`, and `kmax()`

**Comment by benallfree** [comment]
Ready for merge along with https://github.com/adonisjs/adonis-lucid/pull/205. See screen snaps above

**Comment by thetutlage** [comment]
@benallfree 

Can we prefix them with `get`, coz they have to the last method in the chain anyways, so it will be

`getCount()`, `getAvg()`, `getSum()`, `getMin()`, and `getMax()`

**Comment by benallfree** [comment]
Done

**Comment by thetutlage** [comment]
@benallfree Are we good to merge?

**Comment by benallfree** [comment]
Yes, both this and https://github.com/adonisjs/adonis-lucid/pull/205

**Pull request merged**